cmd/flow-dps-rosetta/README.md

```diff
@@ -0,0 +1,27 @@
+# Flow DPS Rosetta
+
+## Description
+
+The Flow DPS Rosetta server implements the Rosetta Data API specifications for the Flow network.
+It uses the Flow DPS Server's GRPC API as the backend to query the required data.
+Flow core contract addresses are derived from the chain ID which which the service is started.
+This allows the Rosetta API to access state remotely, or locally by running the Flow DPS Server on the same host.
+
+## Usage
+
+```sh
+Usage of flow-dps-rosetta:
+  -a, --api string     host URL for GRPC API endpoint (default "127.0.0.1:5005")
```

Comment:
Disagree. It's a host URL. The host is a machine, of which we are the client. That host has a URL. A URL is pretty clearly defined. An address is a more generic name for a URL. It could be the host URL or the host address.